views/includes/page-templates/managing-officer-corporate.html

```diff
@@ -67,7 +67,11 @@ <h2 class="govuk-heading-l">Who can we contact about this managing officer?</h2>
 
       {% include "includes/save-and-continue-button.html" %}
 
-      {% set removePathURL = OE_CONFIGS.MANAGING_OFFICER_CORPORATE_URL + OE_CONFIGS.REMOVE + "/" + id %}
+      {% if pageParams.isRegistration %}
```

Comment:
Do we need it to do this way or follow what we have done in BO , add removePathURL in views/beneficial-owner-individual and views/update/update-beneficial-owner-individual

Reply:
I think since the remove button is already in the form in the page-templates for the MOs, doing it this way would be better, instead of adding the remove button to each separate view rather than in the templates, let me know what you think